packit/cli/utils.py

```diff
@@ -66,6 +67,17 @@ def covered_func(config=None, *args, **kwargs):
                 else:
                     logger.error(exc)
                 sys.exit(exit_code or 2)
+            except GithubException as exc:
+                if config and config.debug:
+                    logger.exception(exc)
+                else:
+                    logger.error(exc)
```

Comment:
This could still be a different issue: e.g. token is set but doesn't have permissions to access selected resource on Github API.

Therefore I'd make the error message more generic, something like: `f"We've encountered an error while talking to GitHub API, please make sure that you pass GitHub API token and it has correct permissions, precise error message: {exc}"` Ideally we should also print a link to https://github.com/packit-service/packit/blob/master/docs/